### ISSUE TITLE
Add complete mute functionality

### ISSUE THREAD

**Issue by cbrxyz** [cbrxyz/pi-bot#322]
Add functionality to do a "complete mute," where users can not send or see messages.

**Comment by Nydauron** [comment]
Would this be just used for those who self mute? Also, might try to implement #325 alongside this.

**Comment by cbrxyz** [comment]
Yeah, I was thinking a flag could be added to the `!selfmute` command to enable a complete selfmute, like `!selfmute 3h --complete`. For #325, I would hope the button would work for all types of self mutes.
